1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts

```diff
@@ -20,10 +20,10 @@ import { isPalindrome } from '.'
 
 describe('isPalindrome tests', () => {
   it('Returns a boolean', () => {
-    expect(typeof isPalindrome()).toBe('boolean');
+    expect(typeof isPalindrome('')).toBe('boolean');
   })
 
   it('Knows that "mom" is a palindrome', () => {
-    expect(typeof isPalindrome('mom')).toBe(true);
+    expect(isPalindrome('mom')).toBe(true);
```

Comment:
**nitpick (non-blocking)**: `.toBe(true)` could be expressed as `.toBeTruthy()`

Reply:
I always forget about this one! Thanks, I will update it